resources/sql/queries.sql

```diff
@@ -207,6 +207,36 @@ WHERE 1=1
 /*~ ) ~*/
 ;
 
+-- :name get-form-template :? :1
+SELECT
+  id,
+  organization,
+  title,
+  start,
+  endt as "end",
+  fields::TEXT,
+  enabled,
+  archived,
+  TRUE AS active -- TODO implement active and archiving
+FROM form_template
+WHERE id = :id;
+
+-- :name get-form-templates
+select id from form_template;
```

Comment:
Dead code?

Reply:
Yeah… We need a proper query to replace the old plural query eventually, but this is not it. I'll remove it.